imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import unicode_literals
+from django.test import TestCase
+import factory
+from django.contrib.auth.models import User
+from faker import Faker as fake_factory
+from models import Photo, Album
+from imager_profile.models import ImagerProfile
+
+fake = fake_factory()
+
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = User
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    email = factory.LazyAttribute(lambda x:
+                                  '{0}@example.com'.format(x.first_name))
+    username = factory.Sequence(lambda n: 'user{}'.format(n))
+
+
+class PhotoFactory(factory.Factory):
+    class Meta:
+        model = Photo
+    title = fake.sentences(nb=1)[0]
+    description = fake.text()
+    file = fake.mime_type()
+
+
+class AlbumFactory(factory.Factory):
+    class Meta:
+        model = Album
+    title = fake.sentences(nb=1)[0]
+    description = fake.text()
+
+
+class PhotoTestCase(TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
You don't want to use this method.  you want to use the `setUp` method.  The problem here is that the instances you create here are held in the database through all tests.  This means that changes you make to individual objects in any test will be persisted between tests.  This will violate the principle of test isolation.